bin/dump_schema.py

```diff
@@ -0,0 +1,13 @@
+import json
```

Comment:
Could you add a quick note in module docstring on what this script is for/what it's doing?